### DISCUSSION TITLE
Android 9 and Screen Capture

### DISCUSSION THREAD

**Discussion by ant0nwax** [rustdesk/rustdesk#2821] [Q&A]
In the docs it is written that Android 6+ is necessary for screen capture. On my Samsung Note 8 the service cannot connect to rustdesk because Audio Capture is not supported only from Android 10+ in RustDesk 1.1.10-1

which is the last rust desk version that supports screen capture without audio capture to be used on Android 9?

thx

**Comment by Heap-Hop** [comment]
The screen capture is independent from audio capture on all Android version. You can just run Android screen capture service without any other services.

**Comment by ant0nwax** [comment]
EDIT:

i had some more time and was debugging:
i found out that it is not the issue of the android app but the mobile data provider, of my home router
https://github.com/rustdesk/rustdesk/issues/376

my workaround: I use a VPN to connect the Note 8 to a different Internet Provider, which is a Fibre Network, from there service connects, I will add information to the issue i linked in above.

Sorry for any inconvenience


thx
service does not connect on my Note 8
it is written that screen capture works only from Android 10. but in docs
it writes from Android 6, so i guess i need an older apk?

Kind Regards From Phoneland
